examples/dht_sensor/dht_sensor.c

```diff
@@ -0,0 +1,39 @@
+/* 
+ *
+ * This sample code is in the public domain.
+ */
+#include "espressif/esp_common.h"
+#include "esp/uart.h"
+#include "FreeRTOS.h"
+#include "task.h"
+#include "dht.h"
+#include "esp8266.h"
+
+/* An example using the ubiquitous DHT** humidity sensors
+ * to print out a new temperature and humidity measurement
+ * every three seconds.
+ */
+int const gpio4 = 4;
+
+void dhtMeasurementTask(void *pvParameters)
+{
+  int8_t temperature = 0;
+  int8_t humidity = 0;
+
+  gpio_set_pullup(gpio4, false, false);

```

Comment:
Ah, ok.. I don't actually have any, but was actually thinking about playing around with those, so maybe I'll get some and do some more experimentation of my own :)  This is definitely fine for now, though..